client/state/sites/reducer.js

```diff
@@ -37,7 +37,10 @@ export function items( state = {}, action ) {
 			} );
 			return indexBy( sites, 'ID' );
 		case DESERIALIZE:
-			return isValidStateWithSchema( state, schema ) ? state : {};
```

Comment:
I just wonder if there is some rule into our guide lines to avoid ternary operator in this context.